portfolio.html

```diff
@@ -16,7 +16,7 @@
 				<section class="careerProfile">
 					
 					<div class="contentHeadings">
-						<i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> 
+						<i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i> 
```

Comment:
inline styles are discouraged. Is this a font awesome requirement or can it be moved into your stylesheet?

Reply:
(assume this comment anywhere you have used an `<i>` tag